src/codecoverage/bot/code_coverage_bot/artifacts.py

```diff
@@ -79,11 +80,18 @@ def download_all(self):
             if taskcluster.is_coverage_task(task)
         ]
 
+        for test_task in test_tasks:
+            while status not in FINISHED_STATUSES:
+                assert status in ALL_STATUSES
+                print('Waiting for task {} to finish...'.format(test_task['status']['taskId']))
```

Comment:
Why not `logger` ?

Reply:
Oops, I copied the code from another project and forgot to change it. Done in 925d08.